node/src/service.rs

```diff
@@ -146,6 +146,7 @@ pub fn new_partial(
 			registry: config.prometheus_registry(),
 			check_for_equivocation: Default::default(),
 			telemetry: telemetry.as_ref().map(|x| x.handle()),
+			compatibility_mode: Default::default(),
```

Comment:
whats this?

Reply:
It's due to [this change](https://github.com/paritytech/substrate/commit/4e1e17cccd499dfe49e8c1bed01957953aa4c839) in Aura.